samples/tab-add-in-combined/nodejs/tab/src/components/App.tsx

```diff
@@ -115,11 +115,32 @@ const App: React.FC = () => {
   const [items, setDiscounts] = useState<Offer[]>([]);
 
   const getData = () => { 
-    fetch('http://localhost:3001/offers', {cache: "no-store"}) //api for the get request
-    .then(response => response.json())
-    .then(data => {
-      console.log(data);
-      setDiscounts(data);})
+    //If you want to run mock APIs with dynamic data, uncomment lines 120 to 124 and comment lines 126 to 142.
+    
+    //fetch('http://localhost:3001/offers', {cache: "no-store"}) //api for the get request
+    //.then(response => response.json())
+    //.then(data => {
+    // console.log(data);
+    //setDiscounts(data);})
```

Comment:
This should be by default, sample data for deployed version only

Reply:
Fixed